test/test_db.py

```diff
@@ -15,7 +15,6 @@
 import pytest
 
 from sopel.db import SopelDB
-from sopel.test_tools import MockConfig
```

Comment:
I requested your review on #1777 mostly because you're the testing expert, but it also works to have you look at that and determine if anything there will need adjusting after the changes here.

(code line unrelated; I just needed an anchor)

Reply:
Oh. I theory no: there might be a merge conflict but nothing I can't fix. The main change here affects the `db` fixture, so it's relatively small.